Lux/src/warp_field.hpp

```diff
@@ -0,0 +1,18 @@
+#include "image.hpp"
+#include "vector_field.hpp"
+
+class warp_field : public image< unsigned int > {
+
+public:
+    warp_field() : image() {}     
```

Comment:
it seems like this constructor and some other functions are being defined twice. it's fine to declare twice but the function body should be in only one place.